src/app/pages/scans/scans-management/base-scan.component.html

```diff
@@ -41,6 +41,9 @@
                     let analyzer of analyzersList[formData.classification]
                   "
                   [value]="analyzer"
+                  disabled="{{ isConfigured(analyzer) }}"
+                  nbTooltip="{{ constructTooltip(analyzer).message }}"
+                  nbTooltipStatus="{{ constructTooltip(analyzer).color }}"
```

Comment:
Instead of calling these functions here individually for each analyzer option, I think it could be better if these changes were made in the `ngOnInit` settings these fields directly on the `analyzersList` attribute.

Reply:
Removed both the functions. For the configured check, we can already access that attribute directly using `analyzerService.rawAnalyzerConfig[analyzer].verification.configured`.
The tooltip part is on hold rn so I have removed those changes.